meta.yml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: rotary
+  version: "0.3.0"
```

Comment:
Change to 0.3.0-beta1 ?

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/commit/d47f9210b734a1c23a5a77cb0bf3029685905e3f

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102